tests/core/string/test_string.h

```diff
@@ -348,22 +348,103 @@ TEST_CASE("[String] Substr") {
 	CHECK(s.substr(3) == "ler Baby");
 }
 
+#define MULTICHECK_STRING_EQ(obj, func, param1, eq) \
```

Comment:
```suggestion
#define MULTICHECK_STRING_EQ(m_obj, m_func, m_param1, m_eq) \
```
Prefix arguments in macros with `m_`